client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -311,32 +311,34 @@ export class PlanFeatures2023Grid extends Component<
 			selectedFeature,
 		} = this.props;
 		return (
-			<div className="plans-wrapper">
-				<QueryActivePromotions />
-				<div className="plan-features">
-					<div className="plan-features-2023-grid__content">
-						<div>
-							<div className="plan-features-2023-grid__desktop-view">
-								{ this.renderTable( planProperties ) }
-							</div>
-							<div className="plan-features-2023-grid__tablet-view">
-								{ this.renderTabletView() }
-							</div>
-							<div className="plan-features-2023-grid__mobile-view">
-								{ this.renderMobileView() }
+			<>
```

Comment:
Could you help me understand the purpose of restructuring DOM elements here? I'm not sure I fully understand it so I'd be grateful.

Reply:
I was thrown off by this change too. Just in case you were thrown off for the same reason @southp try to ignore the whitespace for the diff :)
<img width="664" alt="image" src="https://github.com/Automattic/wp-calypso/assets/3422709/2e1e63d8-cd18-4bcc-b5d5-5f2a2faeba4d">

---
@jeyip the `plans-wrapper` class and div was  technically meant to be a wrapper around the entire plans content. Is there any possibility we can remove the responsiveness embedded in the plans-wrapper div and possibly bring into a child div like `plan-features-2023-grid__content`?

This way we refactor the css and keep the dom-structure intact.

---
Also we definitely need to rebase the work done by Christos and us (#77078) and then work from there. The table elements were extremely challenging to work on and we need to accommodate that complexity. However I agree that it's highly unlikely to clash with the work you have done here 👍

Reply:
> Also we definitely need to rebase the work done by Christos and us (https://github.com/Automattic/wp-calypso/pull/77078) and then work from there.

Great to know -- I'll get started on that 👍